packages/sources/svelte/src/components/actions/VtmnDropdown/VtmnDropdown.svelte

```diff
@@ -0,0 +1,147 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+  import { createEventDispatcher } from 'svelte';
+  import { clickOutside } from '../../../utils/clickOutside';
+
+  /** @restProps */
+
+  /**
+   * Custom classes to apply to the component.
+   * @type {array} options
+   */
+  export let options;
```

Comment:
Good PR!
But I'm not fan to use options. We are limited to the style applied on the element.
For example, if we need to have a skeleton on the content, it is not possible.

Same here, if we want to write something on right
![image](https://user-images.githubusercontent.com/2856778/158979024-ac2b22a2-cda5-495e-a27e-6d96fdbe378b.png)

For me, it's better to have a `VtmnDropdownItems` and do stuff like that : 

```javascript
<VtmnDropdown>
    <VtmnDropdownItem value=""></VtmnDropdownItem>
    <VtmnDropdownItem value="1">Option 1</VtmnDropdownItem>
    <VtmnDropdownItem value="2">Option 2</VtmnDropdownItem>
</VtmnDropdown>
```